KlayGE/Samples/src/CausticsMap/CausticsMap.cpp

```diff
@@ -113,7 +113,7 @@ namespace
 			caustics_map_ = caustics_map;
 		}
 
-		void Pass(uint32_t pass)
+		void Pass(PassType pass)

```

Comment:
We'd better use a new function name, such as CausticsPass. It's not exactly a override case.